Design: 메인페이지UI 구현 #7

LeeHyunji opened the pull request:
## PR 타입(하나 이상의 PR 타입을 선택해주세요)
- [x] 디자인 변경
- [ ] 기능 추가
- [ ] 기능 삭제
- [ ] 버그 수정
- [ ] 의존성, 환경 변수, 빌드 관련 코드 업데이트

## 반영 브랜치
feature/7-mainUI -> main

## 변경 사항
1. 추천음악 / top 200 작업 완료
2. 비슷한 유저 추천 목록
	-  왼쪽 상단에 고정되게 처리
	- user_list_card : 재사용성이 있어보여서 공용 style.css에 작업
3. music_card 업데이트
	- 뮤직 카드 총평점 별점으로 표현 style.css 작업
	- 퍼센트 기준.


## 테스트 결과
변경 사항을 설명하는데 도움이 되는 스크린샷이나 참고 자료를 추가해주세요.
![screencapture-127-0-0-1-5500-index-html-2022-11-04-20_39_34](https://user-images.githubusercontent.com/12287842/199964091-de4fc468-9f00-4bc2-ac58-df164e1bda66.png)

ssuin52 reviewed (approved):
오! main 페이지 만들어주셨네요! 좋습니다:)

LeeHyunji committed:
Design: 메인페이지UI 구현#7

1. 추천음악 / top 200 작업 완료
2. 비슷한 유저 추천 목록
	-  왼쪽 상단에 고정되게 처리
	- user_list_card : 재사용성이 있어보여서 공용 style.css에 작업
3. music_card 업데이트
	- 뮤직 카드 총평점 별점으로 표현 style.css 작업
	- 퍼센트 기준.

Close #7
